.github/workflows/model-evaluation.yml

```diff
@@ -0,0 +1,201 @@
+          AZURE_SPEECH_SERVICE_RESOURCE_GROUP: ${{ vars.AZURE_SPEECH_RESOURCE_GROUP }}
+          AZURE_SPEECH_SERVICE_LOCATION: ${{ vars.AZURE_SPEECH_SERVICE_LOCATION }}
+          AZURE_SPEECH_SERVICE_SKU: ${{ vars.AZURE_SPEECH_SERVICE_SKU }}
+          AZURE_KEY_VAULT_NAME: ${{ vars.AZURE_KEY_VAULT_NAME }}
+          AZURE_USE_AUTHENTICATION: ${{ vars.AZURE_USE_AUTHENTICATION }}
+          AZURE_ENFORCE_ACCESS_CONTROL: ${{ vars.AZURE_ENFORCE_ACCESS_CONTROL }}
+          AZURE_ENABLE_GLOBAL_DOCUMENT_ACCESS: ${{ vars.AZURE_ENABLE_GLOBAL_DOCUMENT_ACCESS }}
+          AZURE_ENABLE_UNAUTHENTICATED_ACCESS: ${{ vars.AZURE_ENABLE_UNAUTHENTICATED_ACCESS }}
+          AZURE_AUTH_TENANT_ID: ${{ vars.AZURE_AUTH_TENANT_ID }}
+          AZURE_SERVER_APP_ID: ${{ vars.AZURE_SERVER_APP_ID }}
+          AZURE_CLIENT_APP_ID: ${{ vars.AZURE_CLIENT_APP_ID }}
+          ALLOWED_ORIGIN: ${{ vars.ALLOWED_ORIGIN }}
+          AZURE_ADLS_GEN2_STORAGE_ACCOUNT: ${{ vars.AZURE_ADLS_GEN2_STORAGE_ACCOUNT }}
+          AZURE_ADLS_GEN2_FILESYSTEM_PATH: ${{ vars.AZURE_ADLS_GEN2_FILESYSTEM_PATH }}
+          AZURE_ADLS_GEN2_FILESYSTEM: ${{ vars.AZURE_ADLS_GEN2_FILESYSTEM }}
+
+        steps:
+          #--------------------------Setup--------------------------------
+
+          - name: Checkout
+            uses: actions/checkout@v4
+
+          - name: Setup python
+            uses: actions/setup-python@v5
+            with:
+              python-version: ${{ matrix.python_version }}
+              architecture: x64
+    
+          - name: Install azd
+            uses: Azure/setup-azd@v1.0.0
+    
+          - name: Install Nodejs
+            uses: actions/setup-node@v4
+            with:
+              node-version: 18
+
+          #--------------------------Deploy--------------------------------
+          - name: Check credentials
+            run: |
+              echo "AZURE_CLIENT_ID: ${{ env.AZURE_CLIENT_ID }}"
+              echo "AZURE_TENANT_ID: ${{ env.AZURE_TENANT_ID }}"
+              echo "AZURE_SUBSCRIPTION_ID: ${{ env.AZURE_SUBSCRIPTION_ID }}"
+              echo "AZURE_PRINCIPAL_ID: ${{ secrets.AZURE_PRINCIPAL_ID }}"
+
+          - name: Azure login
+            uses: azure/login@v2
+            with:
+              client-id: ${{ env.AZURE_CLIENT_ID }}
+              tenant-id: ${{ env.AZURE_TENANT_ID }}
+              subscription-id: ${{ env.AZURE_SUBSCRIPTION_ID }}
+            
+          - name: Azure CLI script
+            uses: azure/cli@v2
+            with:
+              azcliversion: latest
+              inlineScript: |
+                az account show
+
+          #--------------------------Run Evaluation--------------------------------
+          - name: Install dependencies
+            working-directory: ./evaluation
+            run: |
+              python -m pip install --upgrade pip
+              pip install -r scripts/requirements.txt
+              pip install pyrit
```

Comment:
As we've established, this should have been raised in time and we should never have resorted to measures like that. We'll fix the requirements in our project so that we can avoid clashes.

Reply:
I will remove the second line, just make sure that the pip conflict issue is resolved in the evaluation branch.

Reply:
Absolutely, I'm dealing with it!